### ISSUE TITLE
Fetch existing emails on !login

### ISSUE THREAD

**Issue by bpcurse** [JojiiOfficial/Matrix-EmailBridge#29]
Another functional addition would be a !login option to "import" older emails.
This could be expanded by another option defining how far the bot should go back (e.g. the last 25 mails or the last month).

**Comment by JojiiOfficial** [comment]
So you mean a command like
`!import 90` to import 90 last messages 
or
`!import all` to import all emails

**Comment by bpcurse** [comment]
Yes, that would be great.
And maybe even `!import 30d` for 30 days.

**Comment by JojiiOfficial** [comment]
@bpcurse
Does the bridge really need an import command? Isn't it enough if you get asked after logging in into the IMAP account, if you want to import some/all emails from this account?

**Comment by bpcurse** [comment]
Agreed, from my POV/use case it is sufficient to offer this on login only. A separate command shouldn't be needed.
An important option however would be some kind of limiting, as you probably meant with "some/all".
Thank you for considering this feature!
